shell/platform/tizen/tizen_log.h

```diff
@@ -0,0 +1,65 @@
+// Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_TIZEN_LOG_H_
+#define EMBEDDER_TIZEN_LOG_H_
+#include <dlog.h>
+
+#include <cassert>
+#include <cstdlib>
+#include <string>
+
+#ifdef LOG_TAG
+#undef LOG_TAG
+#endif
+#define LOG_TAG "ConsoleMessage"
+
+#undef __LOG
+#define __LOG(id, prio, tag, fmt, arg...)                              \
+  __dlog_print(id, prio, tag, "%s: %s(%d) > " fmt, __FILE__, __func__, \
+               __LINE__, ##arg);
+
+#define __FT_LOG(prio, fmt, args...) \
+  __LOG(LOG_ID_MAIN, prio, LOG_TAG, fmt, ##args)
+
+#define FT_LOGD(fmt, args...) __FT_LOG(DLOG_DEBUG, fmt, ##args)
+#define FT_LOGI(fmt, args...) __FT_LOG(DLOG_INFO, fmt, ##args)
+#define FT_LOGW(fmt, args...) __FT_LOG(DLOG_WARN, fmt, ##args)
+#define FT_LOGE(fmt, args...) __FT_LOG(DLOG_ERROR, fmt, ##args)
+
+#if defined(NDEBUG)
+#define FT_ASSERT(assertion) ((void)0)
+#define FT_ASSERT_NOT_REACHED() ((void)0)
```

Comment:
- How about UNREACHABLE rather than NOT_REACHED?
- What are the differences between these macros (especially FT_ASSET_STATIC vs. FT_COMPILE_ASSERT)? Are they from another project?

Reply:
> * How about UNREACHABLE rather than NOT_REACHED?
 
There is a similar term in [glib](https://developer.gnome.org/glib/stable/glib-Testing.html#g-assert-not-reached).
But your suggestion is okay too, Should I change it?

> * What are the differences between these macros (especially FT_ASSET_STATIC vs. FT_COMPILE_ASSERT)? Are they from another project?

The difference in behavior is that the NDEBUG flag is not considered for FT_COMPILE_ASSERT
These two purposes can be said to be almost the same. but If you always want to check assertions at compile-time, using FT_COMPILE_ASSERT.

Reply:
> Should I change it?

No, you can do as you prefer. I think it's okay.

> These two purposes can be said to be almost the same. but If you always want to check assertions at compile-time, using FT_COMPILE_ASSERT.

I asked because none of them are being used in the code and apparently you copied the macros from somewhere. Do you plan to use the macros in the future?